tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -259,3 +259,97 @@ def test_special_fields(self):
         assert response.data[1]['issue.id'] == groups[1].id
         assert response.data[1]['event_count'] == 2
         assert response.data[1]['user_count'] == 2
+
+    def test_special_field_with_conditions(self):
+        self.login_as(user=self.user)
+        project = self.create_project()
+        self.store_event(
+            data={
+                'event_id': 'a' * 32,
+                'timestamp': self.min_ago,
+                'fingerprint': ['group_1'],
+                'user': {
+                    'email': 'foo@example.com',
+                },
+            },
+            project_id=project.id,
+        )
+        self.store_event(
+            data={
+                'event_id': 'b' * 32,
+                'timestamp': self.min_ago,
+                'fingerprint': ['group_2'],
+                'user': {
+                    'email': 'foo@example.com',
+                },
+            },
+            project_id=project.id,
+        )
+        self.store_event(
+            data={
+                'event_id': 'c' * 32,
+                'timestamp': self.min_ago,
+                'fingerprint': ['group_2'],
+                'user': {
+                    'email': 'bar@example.com',
+                },
+            },
+            project_id=project.id,
+        )
+        self.store_event(
+            data={
+                'event_id': 'd' * 32,
+                'timestamp': self.min_ago,
+                'fingerprint': ['group_3'],
+                'user': {
+                    'email': 'foo@example.com',
+                },
+            },
+            project_id=project.id,
+        )
+        self.store_event(
+            data={
+                'event_id': 'e' * 32,
+                'timestamp': self.min_ago,
+                'fingerprint': ['group_3'],
+                'user': {
+                    'email': 'bar@example.com',
+                },
+            },
+            project_id=project.id,
+        )
+        self.store_event(
+            data={
+                'event_id': 'f' * 32,
+                'timestamp': self.min_ago,
+                'fingerprint': ['group_3'],
+                'user': {
+                    'email': 'foobar@example.com',
+                },
+            },
+            project_id=project.id,
+        )
+
+        groups = Group.objects.all()
+
+        with self.feature('organizations:events-v2'):
+            response = self.client.get(
+                self.url,
+                format='json',
+                data={
+                    'field': ['issue_title', 'event_count', 'user_count'],
+                    'having': ['event_count,>,1', 'user_count,>,1'],
```

Comment:
Open to any ideas around better looking syntax here.

Reply:
Wondering if it makes sense for this to be closer to our `query` syntax? Something like `event_count:>1`

Reply:
Ideally I think this would just be syntax supported in the query parameter and then converted into a having clause by the API. e.g. query could be anything like `environment:production event_count:>1 user.email:something@something.com`

Reply:
We already do automatic conversion to having as part of event/issue search: `times_seen:>1` for example. We'd have to have preset aggregations for this to work.

Reply:
I just noticed we do this already. I'm going to pull it out and refactor. I missed that when reading how the conversion code works. good catch.